docs/notebooks/feature-guides/time-zones.ipynb

```diff
@@ -0,0 +1,344 @@
+      "Inserted 3 rows with 0 errors.\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "UpdateStatus(num_rows=3, num_computed_values=3, num_excs=0, updated_cols=[], cols_with_excs=[])"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from datetime import datetime, timezone\n",
+    "from zoneinfo import ZoneInfo\n",
+    "\n",
+    "naive_dt    = datetime(2024, 8, 9, 23, 0, 0)\n",
+    "explicit_dt = datetime(2024, 8, 9, 23, 0, 0, tzinfo=ZoneInfo('America/Los_Angeles'))\n",
+    "other_dt    = datetime(2024, 8, 9, 23, 0, 0, tzinfo=ZoneInfo('America/New_York'))\n",
+    "\n",
+    "t.insert([\n",
+    "    {'dt': naive_dt,    'note': 'No time zone specified (uses default)'},\n",
+    "    {'dt': explicit_dt, 'note': 'Time zone America/Los_Angeles was specified explicitly'},\n",
+    "    {'dt': other_dt,    'note': 'Time zone America/New_York was specified explicitly'}\n",
+    "])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "ef1f573a-638b-47e1-9420-17d88b2b1bfb",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th>dt</th>\n",
+       "      <th>note</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <td>2024-08-09 23:00:00-07:00</td>\n",
+       "      <td>No time zone specified (uses default)</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>2024-08-09 23:00:00-07:00</td>\n",
+       "      <td>Time zone America/Los_Angeles was specified explicitly</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>2024-08-09 20:00:00-07:00</td>\n",
+       "      <td>Time zone America/New_York was specified explicitly</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>"
+      ],
+      "text/plain": [
+       "                         dt                                               note\n",
+       "0 2024-08-09 23:00:00-07:00              No time zone specified (uses default)\n",
+       "1 2024-08-09 23:00:00-07:00  Time zone America/Los_Angeles was specified ex...\n",
+       "2 2024-08-09 20:00:00-07:00  Time zone America/New_York was specified expli..."
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "t.collect()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2f5a1cd6-6e27-4ea6-9de0-1bf50130c557",
+   "metadata": {},
+   "source": [
+    "Notice that on retrieval, all the timestamps are normalized to the default time zone, regardless of how they were specified during insertion. To represent timestamps in a different time zone, use the `astimezone` method."
```

Comment:
Move the first sentence above the code cell (it's nice to know what to look for in advance).

Reply:
Done.
<!-- Reviewable comment -OCKNGYD2KqYz6sggBdj:b-896fix -->
<!-- Sent from Reviewable.io -->